### PR TITLE
KN-82402 Add new API to Knoema.Client

### PR DESCRIPTION
**Pull request by SlavkaKnoema**

### CODE REVIEW

File: Knoema.Client/Client.cs [hunk 1]
```diff
@@ -605,6 +605,11 @@ public async Task<string[]> UnloadToLocalFolder(PivotRequest request, string des
 			return fileNames;
 		}
 
+		public Task<RawDataFilesModel> GetRawDataFiles(string datasetId)
+		{
+			return ApiGet<RawDataFilesModel>($"/api/1.0/meta/dataset/{datasetId}/raw");
```
Comment (AlexPhilippov) [inline Knoema.Client/Client.cs:610]: I don't get why it's called raw? data/raw is streaming API. And we had a thing called dataset image for source files. Is this different?

File: Knoema.Client/Meta/RawDataFilesModel.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Knoema.Meta
+{
+	public class RawDataFilesModel
+	{
+		public string Dataset { get; set; }
```
Comment (AlexPhilippov) [inline Knoema.Client/Meta/RawDataFilesModel.cs:5]: You send a request with datasetId and then return the same here? Maybe it's unnecessary?

File: Knoema.Client/Client.cs (hunk 1 above)
Comment (SlavkaKnoema) [reply Knoema.Client/Client.cs:610]: it was projected on the Knoema side

File: Knoema.Client/Meta/RawDataFilesModel.cs (hunk 2 above)
Comment (SlavkaKnoema) [reply Knoema.Client/Meta/RawDataFilesModel.cs:5]: With the dataset, the model has a consistent state and also the API was designed at Knoema

**Pull request merged**